plug-ins/groups/class_thief.cpp

```diff
@@ -516,6 +527,7 @@ SKILL_RUNP( steal )
 
         percent  = number_percent( ) + ( IS_AWAKE(victim) ? 10 : -30 );
         percent += victim->can_see( ch ) ? 20 : 0;
+	percent += skill_level_bonus(*gsn_steal, ch);
```

Comment:
percent is used to calculate failure chance, remove this.
add skill_level_bonus to skill variable declaration in the beginning